### ISSUE TITLE
magic spell cool-down indicator

### ISSUE THREAD

**Issue by cfwalter** [ramorris3/alextown#80]
The player should be able to see when a spell is available for casting again

**Comment by MitchTurner** [comment]
I agree.  It's just hard to indicate when you have different spells.  We'll probably going to have to make little icons for each spell and have an indicator for each.  Maybe have a circular meter around each icon?  That's fine tuning, but I think each spell needs independent meters.
